### PR TITLE
Extract_genomic_dna: use newer ucsc-fatotwobit version

### PR DESCRIPTION
**Pull request by mvdbeek**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/extract_genomic_dna/extract_genomic_dna.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<tool id="Extract genomic DNA 1" name="Extract Genomic DNA" version="3.0.3">
+<tool id="Extract genomic DNA 1" name="Extract Genomic DNA" version="3.0.3+galaxy1">
     <description>using coordinates from assembled/unassembled genomes</description>
     <requirements>
         <requirement type="package" version="0.7.1">bx-python</requirement>
```
Comment (nsoranzo) [inline tools/extract_genomic_dna/extract_genomic_dna.xml]:
```suggestion
        <requirement type="package" version="0.8.6">bx-python</requirement>
```

File: tools/extract_genomic_dna/extract_genomic_dna.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
         <requirement type="package" version="1.10.0">six</requirement>
```
Comment (nsoranzo) [inline tools/extract_genomic_dna/extract_genomic_dna.xml]:
```suggestion
        <requirement type="package" version="1.13.0">six</requirement>
```

File: tools/extract_genomic_dna/extract_genomic_dna.xml (hunk 1 above)
Comment (mvdbeek) [reply tools/extract_genomic_dna/extract_genomic_dna.xml]: This isn't compatible with the script, the imports have changed in bx-python.

Comment (nsoranzo) [reply tools/extract_genomic_dna/extract_genomic_dna.xml]: OK, but it would be good to update since 0.7.1 is not Python3-compatible.

**Pull request merged**